.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,185 +2,189 @@ name: Bug Report
+          label: MSAL Logs
+          description: "Please provide [verbose or trace level log messages](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logging.md) from msal. Bug reports without verbose or trace MSAL logs will be closed without comment. Please look at our [Logging Guide](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logging.md) for instructions on how to collect MSAL logs."
+
+    - type: checkboxes
+      attributes:
+          label: Network Trace (Preferrably Fiddler)
+          description: "Please send a [Fiddler trace](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-troubleshoot-network-trace) (strongly preferred) or .har file containing the relevant network requests and responses to msaljshelp@microsoft.com. Your are reponsible for properly sanitizing any network trace you send. **DO NOT post a network trace**, sanitized or not to this or any GitHub issue. GitHub issues for which we don't receive a corresponding network trace e-mail in a reasonable timeframe may be closed without comment."
+          options:
+              - label: Sent
+              - label: Pending
+    - type: textarea
+      attributes:
+          label: MSAL Configuration
+          description: "Please provide your MSAL configuration. Bug reports without MSAL Configuration will be closed without comment."
+          render: javascript
+          placeholder: |
+              {
+                auth: {
+                  clientId: "clientId",
+                  authority: "https://login.microsoftonline.com/common/"
+                },
+                cache: {
+                  cacheLocation: "sessionStorage"
+                  storeAuthStateInCookie: false
+                }
+              }
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Relevant Code Snippets
+          description: "Please provide relevant code snippets that can be used to reproduce this issue."
+          render: javascript
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Reproduction Steps
+          description: "Please provide steps to reproduce or a link to a sample which demonstrates this behavior. If there is no way to consistently reproduce this issue, it may be closed without comment."
+          placeholder: |
+              1.
+              2.
+              3.
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Expected Behavior
+          description: "Please describe what you expect the behavior to be."
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Identity Provider
+          options:
+              - Azure AD / MSA
+              - Azure B2C Basic Policy
+              - Azure B2C Custom Policy
+              - ADFS
```

Comment:
should we spell out what ADFS stands for?

Reply:
I feel like if they use ADFS they know what it means, right?